lib/validation.js

```diff
@@ -134,14 +134,18 @@ function validate (context, request) {
 }
 
 function wrapValidationError (result, dataVar, schemaErrorFormatter) {
+  const FST_ERR_VALIDATION_CODE = 'FST_ERR_VALIDATION'
```

Comment:
Is putting it into a variable faster?

Reply:
@Uzlopak You are right. Hardcoding the values is on average faster. I have updated the draft, commit https://github.com/fastify/fastify/pull/4632/commits/b4b8aaeb023fb219d6310bba39739279220b54e3. I am attaching output from the 5-minute benchmarks for reference.

```shell
# hardcoded

$ npx concurrently -k -s first "node ./examples/benchmark/simple.js" "npx autocannon -c 100 -d 300 -p 10 localhost:3000/"
[1] Running 300s test @ http://localhost:3000/
[1] 100 connections with 10 pipelining factor
[1]
[1]
[1] ┌─────────┬──────┬───────┬───────┬───────┬─────────┬─────────┬────────┐
[1] │ Stat    │ 2.5% │ 50%   │ 97.5% │ 99%   │ Avg     │ Stdev   │ Max    │
[1] ├─────────┼──────┼───────┼───────┼───────┼─────────┼─────────┼────────┤
[1] │ Latency │ 5 ms │ 13 ms │ 21 ms │ 24 ms │ 13.1 ms │ 4.33 ms │ 143 ms │
[1] └─────────┴──────┴───────┴───────┴───────┴─────────┴─────────┴────────┘
[1] ┌───────────┬─────────┬─────────┬─────────┬─────────┬──────────┬─────────┬─────────┐
[1] │ Stat      │ 1%      │ 2.5%    │ 50%     │ 97.5%   │ Avg      │ Stdev   │ Min     │
[1] ├───────────┼─────────┼─────────┼─────────┼─────────┼──────────┼─────────┼─────────┤
[1] │ Req/Sec   │ 70271   │ 72767   │ 73791   │ 75327   │ 73675.95 │ 1915.57 │ 45320   │
[1] ├───────────┼─────────┼─────────┼─────────┼─────────┼──────────┼─────────┼─────────┤
[1] │ Bytes/Sec │ 13.2 MB │ 13.7 MB │ 13.9 MB │ 14.2 MB │ 13.9 MB  │ 360 kB  │ 8.52 MB │
[1] └───────────┴─────────┴─────────┴─────────┴─────────┴──────────┴─────────┴─────────┘
[1]
[1] Req/Bytes counts sampled once per second.
[1] # of samples: 300
[1]
[1] 22104k requests in 300.47s, 4.16 GB read
[1] npx autocannon -c 100 -d 300 -p 10 localhost:3000/ exited with code 0
--> Sending SIGTERM to other processes..
[0] node ./examples/benchmark/simple.js exited with code SIGTERM
```

```shell
# using a variable

$ npx concurrently -k -s first "node ./examples/benchmark/simple.js" "npx autocannon -c 100 -d 300 -p 10 localhost:3000/"
[1] Running 300s test @ http://localhost:3000/
[1] 100 connections with 10 pipelining factor
[1]
[1]
[1] ┌─────────┬──────┬───────┬───────┬───────┬─────────┬─────────┬────────┐
[1] │ Stat    │ 2.5% │ 50%   │ 97.5% │ 99%   │ Avg     │ Stdev   │ Max    │
[1] ├─────────┼──────┼───────┼───────┼───────┼─────────┼─────────┼────────┤
[1] │ Latency │ 6 ms │ 15 ms │ 19 ms │ 21 ms │ 13.4 ms │ 3.94 ms │ 165 ms │
[1] └─────────┴──────┴───────┴───────┴───────┴─────────┴─────────┴────────┘
[1] ┌───────────┬─────────┬─────────┬─────────┬─────────┬──────────┬─────────┬─────────┐
[1] │ Stat      │ 1%      │ 2.5%    │ 50%     │ 97.5%   │ Avg      │ Stdev   │ Min     │
[1] ├───────────┼─────────┼─────────┼─────────┼─────────┼──────────┼─────────┼─────────┤
[1] │ Req/Sec   │ 69631   │ 70591   │ 72127   │ 73279   │ 72053.23 │ 1775.94 │ 44332   │
[1] ├───────────┼─────────┼─────────┼─────────┼─────────┼──────────┼─────────┼─────────┤
[1] │ Bytes/Sec │ 13.1 MB │ 13.3 MB │ 13.6 MB │ 13.8 MB │ 13.5 MB  │ 334 kB  │ 8.33 MB │
[1] └───────────┴─────────┴─────────┴─────────┴─────────┴──────────┴─────────┴─────────┘
[1]
[1] Req/Bytes counts sampled once per second.
[1] # of samples: 300
[1]
[1] 21617k requests in 300.27s, 4.06 GB read
[1] npx autocannon -c 100 -d 300 -p 10 localhost:3000/ exited with code 0
--> Sending SIGTERM to other processes..
[0] node ./examples/benchmark/simple.js exited with code SIGTERM
```